docs/workshop-schedule.md

```diff
@@ -3,39 +3,49 @@ title: Workshop Schedule
+| 10:15 - 10:45 am    | [Project organization](https://docs.google.com/presentation/d/1ncqxXlC0-PGEK-yE7S-nDYnMPhrOUPbI95EJy283wCs/edit?usp=sharing)                                  |
+| 10:45 - 11:00 am    | _Coffee break_                                                                                                                                                |
+| 11:00 - 11:30 am    | [Strategies for data sharing](https://docs.google.com/presentation/d/10NEkqHkAw9KBAbkNV-vtPwLFc9ZHksHEcCFty32efiE/edit?usp=sharing)                           |
+| 11:30 am - 12:00 pm | [Introduction to UNIX and the command line](https://docs.google.com/presentation/d/1WPXkItJZEUXMY20cLrdMXHiBC2PyunR14RUSDg4nfIc/edit?usp=sharing)             |
+|                     | [_Resource_: UNIX Reference](resources/unix_reference.html)                                                                                                   |
+|                     | [_Resource_: UNIX Cheatsheet (pdf)](resources/unix_quick_reference.pdf)                                                                                       |
+| 12:00 - 12:30 pm    | [Introduction to Git; Forking and cloning a repository](https://docs.google.com/presentation/d/1eiGZA4PYBKJx5HDCo3UDOAB7q415gg96TehxilCHwlA/edit?usp=sharing) |
+| 12:30 - 1:30 pm     | _Lunch_                                                                                                                                                       |
+| 1:30 - 2:15 pm      | [Shell scripting](https://docs.google.com/presentation/d/1SDUyYVNgvDDRodVqmDQPVQ5wnjQesWfBTg0EAcdbcSo/edit?usp=sharing)                                       |
+| 2:15 - 2:45 pm      | [The Git stage/commit/push workflow](https://docs.google.com/presentation/d/1_YckNhAkp_82PKR6PGS5SdaKDgoueYVTXaPi5pQV9ik/edit?usp=sharing)                    |
+| 2:45 - 3:00 pm      | _Coffee break_                                                                                                                                                |
+| 3:00 - 3:45 pm      | [Organizing code in scripts and notebooks](https://docs.google.com/presentation/d/1AJr6uQhwLnZfis1wNc_e2XY4XSMEuVscIfAsVgnM5Bk/edit?usp=sharing)              |
+| 3:45 - 4:15 pm      | [Managing packages and environments](https://docs.google.com/presentation/d/1GCbu2F6LeEPOu5DzDsTgwu1__9YDVydvPo911fBG1i0/edit?usp=sharing)                    |
+| 4:15 - 4:45 pm      | [Working with branches in Git](https://docs.google.com/presentation/d/1s7BSHgTSDuXIzI1ROS-JSneB6NXfQVWOec6lhc8eIWA/edit?usp=sharing)                          |
+| 4:45 pm             | _Adjourn_                                                                                                                                                     |
+| 5:30 pm             | _Optional_ Meet in the hotel lobby to Lyft to dinner                                                                                                          |
```

Comment:
Is everyone in the same hotel? Probably not, and some of us will probably drive folks?

Reply:
Maybe just remove this line from the schedule altogether? We're a small enough group that we can just figure this out and communicate it when we adjourn.

Reply:
Agreed about removing. We may even just head straight over from the workshop!